cfme/ansible/credentials.py

```diff
@@ -212,6 +252,97 @@ def before_fill(self, values):
                 continue
 
 
+def machine_credential(credential):
```

Comment:
_Optional_ refactor, shouldn't these all be plural? i.e. `machine_credentials(credentials)`?

Reply:
Okay I will update it.